### PR TITLE
Mark no-wrap and block-quote tags in different colours

### PR DESCRIPTION
**Pull request by 70ray**
Mark no-wrap and block-quote tags in different colours. This addresses issue [#2063](https://www.pgdp.net/c/tasks.php?action=show&task_id=2063)

Sandbox at: https://www.pgdp.org/~rp31/c.branch/ool_colour

### CODE REVIEW

File: tools/proofers/previewStyles.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* exported defaultStyles */
+
+// these are the default values. If the user changes anything the new
+// styles are saved in local storage and reloaded next time.
+// the foreground and background colours for plain text, italic, bold,
+// gesperrt, smallcaps, font change, other tags, highlighting issues
+// and possible issues.
+// An empty color string means use default color
+var defaultStyles = {
```
Comment (chrismiceli) [inline tools/proofers/previewStyles.js:9]: *minor* we could use `const` here.

**Review by chrismiceli — APPROVED** [review]

File: tools/proofers/previewStyles.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/* exported defaultStyles */
```
Comment (cpeel) [inline tools/proofers/previewStyles.js:1]: I'd prefer this remain in `previewControl.js`. Every separate JS file requires another round-trip for the browser which decreases site performance.

Comment (70ray) [reply tools/proofers/previewStyles.js:1]: I tried that at first but another problem occured. I think I can work round it a different way.

**Comment by 70ray** [comment]
I removed the unnecessary addEventListener("DOMContentLoaded") from preview.js. This wasn't really necessary for this purpose but I happened to notice it was not needed. The file can be viewed ignoring white space changes because most of it is unindented.

**Comment by 70ray** [comment]
Sandbox updated again.

**Review by cpeel — APPROVED** [review]

**Review by srjfoo — APPROVED** [review]

**Pull request merged**